examples/pytorch/speech-recognition/train_seq2seq.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+LOG_DIR="./logs"
+OUTPUT_DIR="./outputs"
+mkdir -p $LOG_DIR
+mkdir -p $OUTPUT_DIR
+##    SETTINGS     ## 
+MODEL=$1
+BATCH_SIZE=$2
+device_id=$3
+log_file="${LOG_DIR}/${model_name}.log"
+output_dir="$OUTPUT_DIR/$model_name"
+
+## END OF SETTINGS ##
+
+export TRANSFORMERS_CACHE=/nas/huggingface_pretrained_models
+export HF_DATASETS_CACHE=/nas/common_data/huggingface
+
+args="
```

Comment:
I think it's better to have a separated `.yml` for this, as well as for all other scripts.

Reply:
@yeomni I noted your comments. I will fix them in the next pull requests